biofefi/pages/6_View_Experiments.py

```diff
@@ -52,9 +58,12 @@
         plot_box(data_analysis, "Data Analysis Plots")
     ml_metrics = ml_metrics_path(experiment_path)
     ml_plots = ml_plot_dir(experiment_path)
+    predictions = ml_predictions_path(experiment_path)
     if ml_plots.exists() and ml_metrics.exists():
         display_metrics_table(ml_metrics)
         plot_box(ml_plots, "Machine learning plots")
+        preds = pd.read_csv(predictions)
+        display_predictions(preds)
```

Comment:
Can this be done in a separate and not nested `if` statement? Maybe that would get around the complexity check?